cfme/utils/dockerbot/pytestbase/check_provisioned.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+import datetime
+import sys
+from cfme.utils.conf import env
+
+
+def main():
+    print(datetime.date.today())
+    if env.appliances[0].hostname == "SPROUT_SHOULD_OVERRIDE_THIS":
```

Comment:
**QUESTION:*** Could we have cfme/utils/dockerbot/pytestbase/setup.sh generate a file that has this "SPROUT_SHOULD_OVERRIDE_THIS" value that gets put in the image, so that its not hard coded between the two scripts?

Reply:
There is nothing wrong with the text.   What is wrong is the two scripts share a magic secret between them (not a technical name of course).   It really should be some sort of constant written in one place rather than two (or more).   If you changed it to NULL, it would still be problematic.

Reply:
But if the script that created this image added a file like /initial_hostname or whatever you might call it, the script inside the container could read that file and compare it to the host name.

Reply:
That would work too.   I don't think the original idea was bad though.   Just concerned with the hard coding between the two scripts.

Reply:
I thought about this over the weekend.   It's just not a big deal.  If we looked at a particular file that's still a something both scripts would have to know.   If this was something shared between multiple scripts (i.e. 3 or more) I'd be more concerned.   Sorry for delaying this.

Reply:
i am absolutely certain this has a nicer solution, i just wanted to get a working fix first, i intend to make it pretty later